libs/jquery.oembed.js

```diff
@@ -73,7 +73,7 @@
                                     provider = $.fn.oembed.getOEmbedProvider(data['long-url']);
 
                                     //remove fallback
-                                    if (!!settings.fallback === false) {
+                                    if (settings.fallback === false) {
```

Comment:
Do we get type coercion issues on this if statement?

Reply:
Shouldn't be a problem unless someone is being a dingus.  Default values are sensible.